docs/docs/contributing/tools/sbt.md

```diff
@@ -0,0 +1,11 @@
+# SBT Commands Cheat Sheet
```

Comment:
Merge this page with https://github.com/lampepfl/dotty/blob/master/docs/docs/contributing/workflow.md